chat.schildi.desktop.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
Same, should be in the app's code repo.

Reply:
It already is there, but the execution path needs to be different.
Should the path be replaced via flatpak build commands then?

Reply:
`desktop-file-edit` can be used to modify keys.